module/VuFind/tests/unit-tests/src/VuFindTest/Http/CachingDownloaderTest.php

```diff
@@ -75,27 +92,39 @@ public function testDownload()
         $storage = $this->getMockBuilder(\Laminas\Cache\Storage\StorageInterface::class)
             ->disableOriginalConstructor()
             ->getMock();
-
-        $storage->expects($this->once())->method('hasItem')->with($testCacheKey)->willReturn(false);
-        $storage->expects($this->once())->method('addItem')->with($testCacheKey, $testBody);
-
         $cacheManagerMock = $container->createMock(\VuFind\Cache\Manager::class);
```

Comment:
While we're changing code anyway, the line above could be simplified to use `createMock` as well.

Reply:
I changed a few others as well.